sample/src/App.tsx

```diff
@@ -29,7 +29,6 @@ const reactNavigationV5Instrumentation = new Sentry.ReactNavigationV5Instrumenta
 Sentry.init({
   // Replace the example DSN below with your own DSN:
   dsn: SENTRY_INTERNAL_DSN,
-  debug: true,
```

Comment:
Note: If `close` is called and debug mode is true, then sending any events afterwards would always show an error that the SDK is disabled. Do we want this?

Reply:
@marandaneto What is the behavior on flutter? cc @bruno-garcia

Reply:
on Java and Flutter do nothing because if you call close, the Hub bound to the Sentry static class is NoOp